tests/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name   ="tests"
+version="0.1.0"
+edition="2021"
+
+[dependencies]
+client    ={ workspace=true }
+notary    ={ workspace=true }
+tokio     ={ workspace=true }
+tracing   ={ workspace=true }
+anyhow    ={ version="1.0" }
```

Comment:
nit: Are we moving all deps to workspace crate? If yes, maybe anyhow should be workspace crate?

Reply:
I don't like `anyhow` outside of testing environment. We can use it in workspace though. I could also just strip it out of here too.